bin/conjur-cli.rb

```diff
@@ -251,6 +252,19 @@
       )
     end
   end
+
+  cgrp.desc 'Restart the Conjur server to apply new configuration'
+  cgrp.long_desc(<<~DESC)
+    Performs a phased restart of the puma process, which restarts the worker
+    threads and allows them to pick up any changes to configuration files. Note
+    that this does NOT pick up any changes to environment variables due to Linux
+    process environments being static once a process has started.
+  DESC
+  cgrp.command :apply do |c|
+    c.action do |_global_options, options, _args|
+      Commands::Configuration::Apply.new.call
+    end
```

Comment:
Looks like we're not using any of these block parameters -- can we delete them?

Reply:
We will be shortly so going to leave 'em in.